smac/utils/io/cmd_reader.py

```diff
@@ -10,8 +10,72 @@
 __version__ = "0.0.1"
 
 
-class CMDReader(object):
+help_type = "standard"
+
+
+class ConfigurableHelpFormatter(ArgumentDefaultsHelpFormatter):
+    """
+    Configurable Help Formatter. Can filter out developer options.
+    """
+
+    def __init(self, *args, **kwargs):
+        super(ConfigurableHelpFormatter, self).__init__(*args, **kwargs)
+
+    def _add_item(self, func, args):
```

Comment:
Please add type hints

Reply:
... everywhere

Reply:
Added type hints.
However, _add_item is an internal method of argparse's HelpFormatter class ([https://github.com/python/cpython/blob/6ea20fc719dcaf102d2cefa1786f0421514f7a58/Lib/argparse.py#L233](https://github.com/python/cpython/blob/6ea20fc719dcaf102d2cefa1786f0421514f7a58/Lib/argparse.py#L233)). Its type interface is not defined explicitly and may change in the future.